docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+    ```
+
+* **Enable php 8.3:**
+    ```sh
+    sudo dnf module enable php:remi-8.3 -y
+    ```
+
+* **Install, enable and start php-fpm:**
+    ```sh
+    sudo dnf install php-fpm
+    sudo systemctl enable php-fpm
+    sudo systemctl start php-fpm
+    ```
+
+## Optional step - adding a website
+
+* **Make a user in which the site would belong to and add the needed directories:**
+    ```sh
+    useradd almasite
+    cd /home/almasite/
+    mkdir logs public_html
+    ```
+
+* **Fix the privileges:**
+    ```sh
+    chown almasite:almasite logs public_html
+    ```
+
+* **Create a configuration file in `/etc/httpd/conf.d/` named almasite.conf:**
+    ```sh
+    <VirtualHost *:80>
+        # The domain
+        ServerName almasite.com
+        # The domains alias
+        ServerAlias www.almasite.com
+        # Full path to the website files
+        DocumentRoot /home/almasite/public_html
+        # Full path to the error log file
+        ErrorLog /home/almasite/logs/error.log
+        # A custom log to record requests to the website
+        CustomLog /home/almasite/logs/access.log combined
+        # Apply directives only on a given directory
+        <Directory /home/almasite/public_html>
+            # Enables the server to follow symbolic links within directories
+            Options Indexes FollowSymLinks
+            # Permits the use of .htaccess files to override server configuration settings
+            AllowOverride All
+            # Allow access to the specified directory and its contents to all users
+            Require all granted
+            # Specifies default file for directory access
+            DirectoryIndex index.php
+        </Directory>
+        # Route requests based on patterns
+        ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:9000/home/almasite/public_html/
+    </VirtualHost>
+    ```
+    Notice the comments explaining each directive.
+
+* **Add a php-fpm pool**
+    ```sh
+    cp /etc/php-fpm.d/www.conf /etc/php-fpm.d/almasite.conf
+    ```
+
+    Open the newly created file with your preferred text editor and change the following:
+    - **\[www\]** - with the pools name (.e.g almasite)
+    - **user = apache** - your user (.i.e almasite)
+    - **group = apache** - your group (.i.e almasite)
+    - **listen = /run/php-fpm/www.soc** - your listening method, in this case we use port 9000 (listen = 9000)
+
+* **Add the websites files: (or file in this case ;))**
+    ```sh
+    touch /home/almasite/public_html/index.php
+    ```
+
+    Open your preferred text editor and add the following:
+    ```html
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>Hello world!</title>
+        </head>
+        <body>
+            <?php $now = date("H:i:s", time()); echo "<h1>Hello world!<br>This message is sent from PHP, the time now is $now</h1>" ?>
+        </body>
+    </html>
+    ```
+
+* **Manipulate your hosts file to view your newly created website:**
```

Comment:
This is fine, but keep in mind the intended audience for this post. It might be worth mentioning that this is optional, as they can visit localhost, which has far less friction.